src/transformers/generation/utils.py

```diff
@@ -1945,69 +1940,9 @@ def _contrastive_search(
             [`~generation.GenerateDecoderOnlyOutput`] if `model.config.is_encoder_decoder=False` and
             `return_dict_in_generate=True` or a [`~generation.GenerateEncoderDecoderOutput`] if
             `model.config.is_encoder_decoder=True`.
-
-        Examples:
```

Comment:
function not public -> let's remove the example (preparing its inputs will be more challenging now, as we no longer have API guarantees)